tests/test-stats/test_nbinom_rvs.py

```diff
@@ -11,9 +11,9 @@
 sess = tf.Session()
 
 
-def _test(n, p, size):
-    val_est = nbinom.rvs(n, p, size=size).shape
-    val_true = (size, ) + np.asarray(n).shape
+def _test(n, p, n):
```

Comment:
For `test-stats/` and the `edward.stats` library in general, I think we can leave it to be called `size`. That is, `edward.stats` follows the convention of `scipy.stats`. `edward.models` uses `n` because `edward.models` follows the convention of `tf.contrib.distributions`.